src/ast.rs

```diff
@@ -5,28 +5,23 @@ use std::collections::HashMap;
 #[derive(Clone, Debug, PartialEq)]
 pub struct Tag {
     pub name: String,
+    pub properties: Vec<String>,
```

Comment:
To modeling `extern(c)` have to add `properties` to store `c`.
Therefore `extern(c)` would become:

```rust
Tag {
  name: "extern".to_string(),
  properties: vec!["c".to_string()]
}
```